docs/packaging.rst

```diff
@@ -225,32 +225,77 @@ src/snek5000_canonical/Snakefile
 
 .. code-block:: python
 
-   import os
-
    import snek5000
```

Comment:
I changed something similar following your suggestion. There are 2 reasons I use methods of `Output` instead of pure functions:
1. I would like to load the config stored in `etc` sub package in the phill example.
2. Extra compiler flags are defined as properties of Output, so it is convinient to update config via a method.